pinot-query-planner/src/test/resources/queries/GroupByPlans.json

```diff
@@ -13,6 +13,20 @@
           "\n"
         ]
       },
+      {
+        "description": "SQL hint based group by optimization with select and multiple aggregations one 1 column",
```

Comment:
The test query doesn't have the hint. Did you mean to add the hint to this query ?